dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -276,6 +281,22 @@ def change_config(python, backend, cheatsheet, asciiart):
         console.print(f'[blue]Backend cached_value {backend}')
 
 
+@option_verbose
+@main.command(name="static-check")
+@click.option('-t', '--type', type=click.Choice(PRE_COMMIT_LIST), multiple=True)
+def static_check(verbose: bool, type: Tuple[str]):
+    if check_package_installed('pre_commit'):
+        command_to_execute = ['pre-commit', 'run']
+        for single_check in type:
+            command_to_execute.append(single_check)
+        command_to_execute.append(EXTRA_STATIC_CHECK_OPTIONS)
+        output = run_command(
```

Comment:
I think we should also enable "forwarding unknown options" in this command. This way we could pass any other option of "pre-commit run" directly to it: 
See https://click.palletsprojects.com/en/8.0.x/advanced/#forwarding-unknown-options

What it should look like - we should be able to do:

```
./Breeze2 static-check --type mypy --from-ref HEAD^^^^ --to-ref HEAD^^^
```

To check the 4-th commit back.

Reply:
This also can allow `--files` option to run the test on specific files.

Reply:
Good point. I think it won't work that nicely as I think click does not have "file" option (they have file arguments). And `ignore_unknown_options` will allow us to pass `--file` transparently. And autocomplete works in the way that when you start typing file name I think it will auto-complete the files available. 

But it might be good indeed to have  a `--file` as option for autocomplete. Something to test.

Reply:
@potiuk how do you think we can enable autocomplete if we pass `--file` as option?

Reply:
@potiuk i tried the below code:
```
@option_verbose
@main.command(
    name="static-check",
    context_settings=dict(
        ignore_unknown_options=True,
        allow_extra_args=True,
    ),
)
@click.option('--all-files', is_flag=True)
@click.option('--show-diff-on-failure', is_flag=True)
@click.option('--last-commit', is_flag=True)
@click.option('-t', '--type', type=click.Choice(PRE_COMMIT_LIST), multiple=True)
@click.option('--files')
@click.argument('precommit_args', nargs=-1, type=click.UNPROCESSED)
def static_check(
    verbose: bool,
    all_files: bool,
    show_diff_on_failure: bool,
    last_commit: bool,
    type: Tuple[str],
    files,
    precommit_args,
):
```
If I try to pass command like `Breeze2 static-check --files tests/*`,  only one of the file is passed into click option for files while all other files are passed as unknown argument in the tuple precommit_args. Could you share your views on this?

Reply:
I think we can pass `--filles` as `boolean` flag (without any argument) and then add `--files` we can simply pass transparently all remaining "unknown" arguments "as_is". 

This will be a little cheating but it should work.... So if we run:

```
Breeze2 static-check mypy --files tests/*
```

The run command would look like:

`pre-commit run mypy --files tests/__init_.py tests/always ....`

So the only difference will be that `--files` will be "known" to auto-complete and users will get explanation about it :) - but we will not expect and value after it

Reply:
This is a bit of cheating, but I think it will do the job :).

Reply:
yes it will work, but I am just worried about the order of unknown options in tuple. but as tuple preserves the insertion order it should work.